Presentations/03-data-wrangling.Rmd

```diff
@@ -605,7 +666,7 @@ name: variables
 
 # Creating new variables
 
-We use `mutate()` to create new variables. For example:
+We use `dplyr::mutate()` to create new variables. For example:
```

Comment:
make sure to explain that the `::` notation means that you're using a function from a package

Reply:
Good point!  I will